sources/engine/Xenko.Input/Simulated/InputSourceSimulated.cs

```diff
@@ -15,11 +15,14 @@ public class InputSourceSimulated : InputSourceBase
         private List<GamePadSimulated> gamePads = new List<GamePadSimulated>();
         private List<MouseSimulated> mice = new List<MouseSimulated>();
         private List<KeyboardSimulated> keyboards = new List<KeyboardSimulated>();
-        
+        private List<PointerSimulated> pointerDevices = new List<PointerSimulated>();
+
+
         public IReadOnlyList<KeyboardSimulated> Keyboards => keyboards;
         public IReadOnlyList<MouseSimulated> Mice => mice;
         public IReadOnlyList<GamePadSimulated> GamePads => gamePads;
-        
+        public IReadOnlyList<PointerSimulated> PointerDevices => pointerDevices;
```

Comment:
If we want to follow name convention of other devices, maybe `PointerDevices` should be renamed `Pointers`.